src/ManageDatesView.elm

```diff
@@ -33,44 +33,52 @@ view model =
             model.pickedDates
                 |> List.indexedMap
                     (\i d ->
-                        div []
-                            [ input
-                                [ class "form-control pikaday-input"
-                                , id ("pikaday-instance-" ++ (toString i))
-                                , value (displayDateWithoutTime d)
-                                , disableInput d
+                        div [ class "form__question-section form__question-section--table" ]
+                            [ div [ class "form__question-sub-section form__question-sub-section--table" ]
+                                [ label [ class "form__label" ]
+                                    [ text "Date" ]
+                                , input
+                                    [ class "form__input  pikaday-input"
+                                    , id ("pikaday-instance-" ++ (toString i))
+                                    , value (displayDateWithoutTime d)
+                                    , disableInput d
+                                    ]
+                                    []
                                 ]
-                                []
-                            , button
-                                [ onClick (DeleteDate d)
-                                , style [ ( "margin-left", "0.2rem" ) ]
-                                , disableInput d
+                            , div [ class "form__question-sub-section form__question-sub-section--table form__question-sub-section--button" ]
+                                [ button
+                                    [ onClick (DeleteDate d)
+                                    , disableInput d
+                                    , class "button button--secondary icon icon--bin"
+                                    ]
+                                    []
                                 ]
-                                [ text "Delete" ]
                             ]
                     )
 
         column1 =
-            div [ class "form-group" ]
-                [ div [ class "input-group" ]
+            div []
+                [ span [ class "form__hint" ]
+                    [ i [ class "icon icon--warning" ] [], text "You will be unable to change any dates that have sessions" ]
```

Comment:
please can you add an `icon--margin-right` modifier that adds:
```css
margin-right: 0.25rem;
```
just to give a bit of breathing room between icon and text